### PR TITLE
I_FatalError: Show Qt/Gtk message window on Linux

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
Cf. https://forum.zdoom.org/viewtopic.php?f=15&t=57880

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
I used that initially, but somehow can't get it to work anymore. Should this only be implemented with the SDL message window, or just use it as a fallback? It didn't look as polished.

**Comment by alexey-lysiuk** [comment]
The idea is to use platform/OS/DE specific code as little as possible if good crossplatform replacement is available.
But if Linux users will agree that native GTK window is better I'm fine with it.

Also, could you please stop polluting CI build queue with your pushes?
Enable Travis and AppVeyor for your fork, experiment there, and create PR when it's done (and only after successful compilation of all targets).
Honestly I have a temptation to close this PR to just to free the queue.

File: src/posix/sdl/i_system.cpp [hunk 1]
```diff
@@ -157,6 +158,29 @@ bool gameisdead;
...
+		FString cmd;
+		cmd << "kdialog --title \"" GAMESIG " ";
+		cmd << GetVersionString() << ": No IWAD found\" ";
+		cmd << "--msgbox \"" << errortext << "\"";
+		popen(cmd, "r");
+	} else if (I_GtkAvailable())
+	{
+		Linux_I_FatalError_Gtk(errortext);
+	} else {
+		SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, (std::string("") + GAMESIG + ": No IWAD found").c_str(), errortext, NULL);
```
Comment (edward-san) [inline src/posix/sdl/i_system.cpp]: I don't think you should use `std::string` here. You should do the same thing with `FString` in the `KDE` branch above.

Comment (Alexander-Wilms) [reply src/posix/sdl/i_system.cpp]: Done

**Comment by Alexander-Wilms** [comment]
The build passed for me: https://travis-ci.org/Alexander-Wilms/test/builds/333435742

**Comment by alexey-lysiuk** [comment]
Merged in 55c6a14059717992a04a9e21ee637d0f24ac5292 as one commit after minor cleanup.

**Pull request closed without merging**